tests/installation/setup_zdup.pm

```diff
@@ -19,8 +19,8 @@ use utils;
 sub run() {
     my ($self) = @_;
 
-    $self->wait_boot(ready_time => 600);
-
+    assert_screen 'displaymanager', 500;
+    ensure_unlocked_desktop;
```

Comment:
please don't exchange `wait_boot` for more custom code. We need special handling depending on architecture, machines, etc. It should have been working in before so why is the change necessary?